File: app/src/main/java/com/munin/music/utils/ActivityUtils.java
package com.munin.music.utils;

import android.app.Activity;
import android.content.Context;
import android.content.pm.ActivityInfo;
import android.view.KeyEvent;

public class ActivityUtils {
    public static void changeOrientation(Context context, boolean isLandScape) {
        Activity activity = null;
        if (context instanceof Activity) {
            activity = (Activity) context;
        }
        if (activity == null) {
            return;
        }
        if (isLandScape) {
            activity.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
            return;
        }
        activity.setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
    }
}

File: app/src/main/java/com/munin/music/utils/ActivityUtils.kt
package com.munin.music.utils

import android.app.Activity
import android.content.Context
import android.content.pm.ActivityInfo
import android.view.KeyEvent

object ActivityUtils {
    fun changeOrientation(context: Context, isLandScape: Boolean) {
        var activity: Activity? = null
        if (context is Activity) {
            activity = context
        }
        if (activity == null) {
            return
        }
        if (isLandScape) {
            activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
            return
        }
        activity.requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_PORTRAIT
    }
}
